docs/commandline/pouch_images.md

```diff
@@ -4,12 +4,21 @@ List all images
 
 ### Synopsis
 
-List all images
+List all images in Pouchd.This is useful when you wish to have a look at images and Pouchd will show all local images with their NAME and SIZE.All local images will be shown in a table format you can use.
 
 ```
 pouch images [flags]
 ```
 
+### Examples
+
+```
+$ pouch images ls
```

Comment:
@Starnop 
This part is auto generated by cobra. I think we introduce the wrong command, since it should be `pouch images`, without `ls`.

Here is the code : 
```
	i.cmd = &cobra.Command{
		Use:   "images",
		Short: "List all images",
		Long:  imagesDescription,
		Args:  cobra.NoArgs,
		RunE: func(cmd *cobra.Command, args []string) error {
			return i.runImages(args)
		},
		Example: imagesExample(),
	}
```

`Args:  cobra.NoArgs`

Reply:
OK, I know that, thank you.